embc-app/Controllers/ReferralsController.cs

```diff
@@ -6,6 +6,7 @@
 using Microsoft.AspNetCore.Mvc;
 using System.Collections.Generic;
 using System.Linq;
+using System.Net;
```

Comment:
Is this used in the code?

Reply:
No that should have been removed in my sweep